language/English/strings.po

```diff
@@ -592,6 +600,30 @@ msgctxt "#32218"
 msgid "Maximum supported protocol"
 msgstr ""
 
+msgctxt "#32220"
+msgid "Bad password!"
+msgstr ""
+
+msgctxt "#32221"
+msgid "The entered password is too weak.[CR]SSH root password is unchanged."
+msgstr ""
+
+msgctxt "#32222"
+msgid "Password changed!"
+msgstr ""
+
+msgctxt "#32223"
+msgid "The SSH password for root has been successfully changed by root."
+msgstr ""
+
+msgctxt "#32224"
+msgid "Unknown error!"
+msgstr ""
+
+msgctxt "#32225"
+msgid "There was an error during the process.[CR]SSH root password is unchanged."
```

Comment:
"There was an error. [CR]SSH password has not been changed."

Reply:
OK